frontend/app/components/encryptable/attribute-field.js

```diff
@@ -0,0 +1,98 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import { capitalize } from "@ember/string";
+
+export default class AttributeField extends Component {
+  @service notify;
+  @service intl;
+  @service store;
+  @service clipboardService;
+
+  @tracked
+  isAttributeVisible = this.args.visibleByDefault;
+
+  HIDE_TIME = 5;
+
+  hideCountdownTime;
+
+  hideTimerInterval;
+
+  constructor() {
+    super(...arguments);
+  }
+
+  @action
+  showValue() {
+    if (this.args.row) {
+      this.fetchEncryptable();
+      this.startHideAttributeTimer();
+    }
+
+    this.isAttributeVisible = true;
+  }
+
+  startHideAttributeTimer() {
+    this.hideCountdownTime = new Date().getTime();
+
+    this.hideTimerInterval = setInterval(() => {
```

Comment:
does this timer work on your machine?
once i revealed a secret, it's just being displayed forever